### DISCUSSION TITLE
Set media track

### DISCUSSION THREAD

**Discussion by mss1010** [timdorr/tesla-api#663] [Q&A]
Hi, I see thatt there are several media commands like next track, previous track,...
Do you see any chance that there is a command to set a specified track for playback? Like, "play song x by artist y", or "play TrackID 123"?

**Comment by core-hacked** [comment]
# No
Unless Tesla decides to add this into the app, which looks unlikely, since you don't really need to change the track to a specific song from the App (who's gonna use that?). It would also mean significant modification to their software and a lot of changes, to support a small user base.

**Reply by kribensa** [reply]
It might happen, if only for the Chinese market.
Many Chinese owners have chauffeurs and therefore sit in the back. It’s the reason the early MS had the executive seating option
So having more control over the media via the app may be something they consider if that market requests it

**Reply by core-hacked** [reply]
They are not going to care... To put it simply, if you have a chauffeur you are rich enough to afford a Palladium MX/S with the rear display or put in an aftermarket accessory like we have seen. It is not needed as it is too nieche and requires too much resources to implement. Especially since no groundwork has been layed out. It would require the vehicle to make seperate requests to the Spotify etc. API's and then sync it to the media player. It is not needed and you can just as well get a mercedes if you really need to change music from the back seat. Plus you can just ask the chaffeur to do it.
